docs/advanced-topics/dockstore-cli/yaml-command-line-validator-tool.rst

```diff
@@ -0,0 +1,122 @@
+Yaml Command Line Validator Tool
```

Comment:
1. Suggestion -- it's a .dockstore.yml command-line, not a generic yaml validator tool. Not sure exactly how to phrase it in a title, give the annoying period in `.dockstore.yml`.
2. It feels like it should be command-line, with a hyphen. Googled briefly, and the found this [Microsoft guide](https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/command-line). I guess it's an adjective in this case, which would make it hyphenated.

Reply:
For point 2, I was going off the style guide
https://github.com/dockstore/dockstore-documentation/blob/784e95bd05328d94408f177c7d632edf78b5bcaa/style-guide.md?plain=1#L89

I am happy to change though.

Reply:
Whoops I did not know I was contradicting the style guide. @aofarrel , any thoughts on my above comment and the MS guide (simplified: command-line as an adjective, command line as a noun)?

But Flynn, you should stick with our style guide unless there's agreement to change it.

Reply:
IMHO, the commandline/command line/command-line thing only matters in the sense that it should be consistent. Some style guides ([Microsoft for instance](https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/command-line)) do make the adjective/noun distinction you do, but my intent when writing that was full consistency. Would be open to changing that part of the style guide though.

Reply:
Point 1, I changed the name to :

> Dockstore Yaml Command Line Validator Tool